spec/features/users/new_user_registration_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "New User Registration Spec" do
   before :each do
-    # @items = create_list(:item, 5)
+    @user = build(:user)
```

Comment:
Factory Bot gem 
It would be the same as: 
`@user = User.new(name: "", address: "", city: "", state: "", etc....`

Reply:
Build is not in db while 'create' is!!